src/Components/GradienBgSection.js

```diff
@@ -45,13 +48,17 @@ const GradienBgSection = props => {
     return (
       <Container>
         <section className="content center">
-        <Text>Top investment deals are impossible to get for the most of us.<p></p>
-
-We are changing this by making the best wealth creation strategies accessible to everyone, with no minimum.
+        <Text>
+          <strong>Top investment deals are impossible to get for the most of us.</strong>
+          <br></br><br></br>
 
-We use a combination of derivatives, synthetic assets and cryptocurrencies to mirror battle proof hedge funds.<p></p>
-<A href="#">Lightpaper</A>
-  </Text>
+          For example, Ray Dalio used to accept money from investors who had a net worth of at least <strong>$5 billion</strong> and with a minimum investment requirement of <strong>$100 million</strong>. 
+          <br></br>Nowadays he won’t accept any new investors.
+          <br></br><br></br>
+          We are making the best asset allocation strategies accessible to everyone on the Ethereum network with no minimum.
```

Comment:
This claim is quite bold. A strategy that has historically good performance is not guaranteed to have the "best" performance when the economical climate changes